File: llmio-node/src/consts.ts
import type { Style } from "./types.js";

export const Version = "dev";

export const DefaultPort = "7070";

export const KeyPrefix = "sk-llmio-";

export const BalancerLottery = "lottery";
export const BalancerRotor = "rotor";
export const BalancerDefault = BalancerLottery;

export const StyleOpenAI: Style = "openai";
export const StyleOpenAIRes: Style = "openai-res";
export const StyleAnthropic: Style = "anthropic";
export const StyleGemini: Style = "gemini";

File: consts/consts.go
package consts

type Style = string

const (
	StyleOpenAI    Style = "openai"
	StyleOpenAIRes Style = "openai-res"
	StyleAnthropic Style = "anthropic"
	StyleGemini    Style = "gemini"
)

const (
	// 按权重概率抽取，类似抽签。
	BalancerLottery = "lottery"
	// 按顺序循环轮转，每次降低权重后移到队尾
	BalancerRotor = "rotor"
	// 默认策略
	BalancerDefault = BalancerLottery
)

const (
	KeyPrefix = "sk-github.com/atopos31/llmio-"
	KeyLength = 32
)
